Code/Framework/AzFramework/AzFramework/Application/Application.cpp

```diff
@@ -213,7 +213,9 @@ namespace AzFramework
 
     void Application::StartCommon(AZ::Entity* systemEntity)
     {
-        m_pimpl.reset(Implementation::Create());
+        
+        auto implementationFactory = AZ::Interface<Application::ImplementationFactory>::Get();
+        m_pimpl = (implementationFactory != nullptr) ? implementationFactory->Create() : nullptr;
```

Comment:
super nitpick: extra line here.
```suggestion
        auto implementationFactory = AZ::Interface<Application::ImplementationFactory>::Get();
        m_pimpl = (implementationFactory != nullptr) ? implementationFactory->Create() : nullptr;
```